src/engine/renderer/glsl_source/cameraEffects_fp.glsl

```diff
@@ -38,7 +38,7 @@ void	main()
 	// calculate the screen texcoord in the 0.0 to 1.0 range
 	vec2 st = gl_FragCoord.st / r_FBufSize;
 
-	vec4 color = clamp(texture2D(u_CurrentMap, st), 0.0, 1.0);
+	vec4 color = texture2D(u_CurrentMap, st);
 
 	color.rgb *= u_LightFactor;
 
```

Comment:
I guess the second clamp should now be 4-dimensional instead of only rgb.

Reply:
Right! 😄️